### ISSUE TITLE
Batch Normalization doesn't function

### ISSUE THREAD

**Issue by IsHYuhi** [IsHYuhi/Image_Inpainting_Using_Partial_Convolutions#2]
In line 55 to 56, bn hasn't defined. 
```
        if hasattr(self, 'bn'):
            output = self.batch_norm(output)
```

change 'bn' to 'batch_norm'
